lib/utils/env_vars.js

```diff
@@ -0,0 +1,17 @@
+require('dotenv').config();
+
+const env_vars = {
+  PLAYGROUND_ID: process.env.PLAYGROUND_ID,
+  PLAYGROUND_EMAIL: process.env.PLAYGROUND_EMAIL,
+  PLAYGROUND_PASSWORD: process.env.PLAYGROUND_PASSWORD,
+  BASE_URL: process.env.BASE_URL,
+  BRANCH_NAME: process.env.BRANCH_PATH && process.env.BRANCH_PATH.split('/')[1] || 'test',
+  REPO_URL: process.env.REPO_URL,
+  DEBUG: process.env.DEBUG,
+};
```

Comment:
These `env_vars` just mirror `process.env` variables, so I'm not sure I understand why we're using the former instead of the latter. It would be helpful to have a comment somewhere in here about why this is preferable to using the `process.env` versions.

Reply:
Does the new comment make sense?